From d167df1cb2f11c399ebca2043d91e81a43d4e0fb Mon Sep 17 00:00:00 2001
From: Raphael Gallais-Pou <raphael.gallais-pou@foss.st.com>
Date: Wed, 13 Sep 2023 14:33:00 +0200
Subject: [PATCH 0587/1141] arm64: dts: st: add LTDC reference clock on
 stm32mp253

LTDC has a reference clock from RCC. Add it.

Signed-off-by: Raphael Gallais-Pou <raphael.gallais-pou@foss.st.com>
Change-Id: I52a36a459062d7e48af49d40bc1207c5d565c88d
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/330815
Reviewed-by: Yannick FERTRE <yannick.fertre@foss.st.com>
Domain-Review: Yannick FERTRE <yannick.fertre@foss.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
---
 arch/arm64/boot/dts/st/stm32mp253.dtsi | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

--- a/arch/arm64/boot/dts/st/stm32mp253.dtsi
+++ b/arch/arm64/boot/dts/st/stm32mp253.dtsi
@@ -137,6 +137,6 @@
 };
 
 &ltdc {
-	clocks = <&rcc CK_BUS_LTDC>, <&syscfg 0>;
-	clock-names = "bus", "lcd";
+	clocks = <&rcc CK_BUS_LTDC>, <&rcc CK_KER_LTDC>, <&syscfg 0>;
+	clock-names = "bus", "ref", "lcd";
 };
